spec/inputs/kafka_spec.rb

```diff
@@ -13,6 +14,16 @@ def queue_event(msg, output_queue)
 end
 
 
+class TestKafkaGroup < Kafka::Group
+  def run(a_num_threads, a_queue)
+    a_queue << 'Kafka message'
+  end
+  private
+  def create_consumer_group(options)
```

Comment:
since you are now doing 
```ruby
expect(kafka).to receive(:create_consumer_group) do |options|
  TestKafkaGroup.new(options)
end
```

this `create_consumer_group` method is not required in the `TestKafkaGroup` no?